ansible/roles/distributed_press/templates/nginx-static.j2

```diff
@@ -12,26 +12,36 @@ server {
 
   error_page 404 /404.html;
 
+  add_header Cache-Control no-cache;
   add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload" always;
-  add_header 'X-Frame-Options' 'ALLOW-FROM *';
+  add_header X-Frame-Options "sameorigin";
+  add_header X-XSS-Protection "1; mode=block";
```

Comment:
I think we want to support XSS for stuff like p2p web apps

Reply:
yikes!